src/atomicassets.cpp

```diff
@@ -229,14 +325,15 @@ ACTION atomicassets::remcolauth(
 *  This will make the account get notified on every relevant action concerning this collection using require_recipient()
 *  NOTE: It will consequently allow the account to make any of these actions throw (fail).
 *        Only add trusted accounts to this list
+*  Limit of 24 notify accounts to allow low level reading of collections row for CPU optimizations
```

Comment:
How did you arrive at 24 as the limit? Have we checked existing collections on the various chains to see if any have more than 24?

Reply:
24 is just an arbitrary limit. It can be higher, but that becomes more of a burden on CPU. 

Any entries beyond the limit of 24 are simply not read by the contract while performing auth checks, so it'll behave as if they're not authorized & would remain safe. They would, however, be read & updated when adding or removing authorized editors / notification recipients, so it should be just fine in that regard.

Lets say you have an ecosystem of smart contracts that you want to add as authorized editors; the limit of 24 can be problematic in that scenario, but an easy solution is to just have 1 account as the authorized editor & allow all of the other contracts within your ecosystem to have @eosio.code active permissions to execute on it's behalf.